org.cloudcompare.CloudCompare.appdata.xml

```diff
@@ -3,9 +3,9 @@
   <id>org.cloudcompare.CloudCompare</id>
   <name>CloudCompare</name>
   <releases>
-    <release date="2022-07-15" version="2.12.4"/>
+    <release date="2023-06-15" version="2.13.beta1"/>
   </releases>
-  <project_license>GPL-2.0</project_license>
+  <project_license>GPL-3.0</project_license>
```

Comment:
Do you think it would be possible to move this file to the main repo and install it as part of the usual installation? Then we wouldn't need to maintain it here in the flatpak.

Reply:
Yes it would be possible I guess, but I'm not sure how it would be easier to maintain in this case.

Reply:
Well, then we don't have to sync the file with the source version. This is better done in the source repo. When the version is updated, the file can be updated in the same PR.

Even better, you can add variables and automatically install this file with the correct version. See this PR of mine for the QGroundControl project: https://github.com/mavlink/qgroundcontrol/pull/10751/files. When the XML file is installed, `@REL_DATE@` and `@REL_VERSION@` are replaced in CMake with information from git. This way, you do not have to touch the file the next time the version is updated.

Reply:
Thanks, yes we need to think about that. But anyway for now I think it's better to keep this file into the flatpak repo. I will integrate this in the next dev cycle.